packages/ui/tsconfig.json

```diff
@@ -5,7 +5,7 @@
     }
   },
   "compilerOptions": {
-    "target": "es5",
+    // "target": "es5",
```

Comment:
Otherwise there'd be error

```
tsup
✘ [ERROR] Transforming const to the configured target environment ("es5") is not supported yet

    tsup.config.ts:1:0:
      1 │ const __injected_filename__ = "/Users/zhamujun/Amplify-workpl...
        ╵ ~~~~~

  The target environment was set to "es5" here:

    tsconfig.json:8:14:
      8 │     "target": "es5",
        ╵               ~~~~~
```